docs/source/security/cors.mdx

```diff
@@ -197,26 +222,42 @@ The most important rule for whether or not a request is ["simple"](https://devel
 
 However, Apollo Server also handles [`GET` requests](../requests#get-requests). `GET` requests do not require a `Content-Type` header, so they can potentially be simple requests. So how can we ensure that we only execute `GET` requests that are _not_ simple requests? If we require the request to include an HTTP header that is never set automatically by the browser, then that is sufficient: requests that set HTTP headers other than the handful defined in the spec must be preflighted.
 
-Apollo Server 3.7 introduced a CSRF prevention feature, which you can enable by passing `csrfPrevention: true` to `new ApolloServer()`. When this feature is enabled, Apollo Server only executes GraphQL operations if at least one of the following conditions is true:
+Apollo Server 3.7 introduced a CSRF prevention feature, which can be enabled in versions of Apollo Server 3.7+ by passing `csrfPrevention: true` to the `ApolloServer` constructor.
```

Comment:
Doesn't need to talk about 3.7, doesn't need to be enabled any more

Reply:
I originally left this section in for when this article is eventually on the main branch people are viewing. But the more I think about it, the more agree with you. If folks want AS3 info, they can navigate to that branch.